lametro/models.py

```diff
@@ -487,6 +487,25 @@ class LAMetroEvent(Event, LiveMediaMixin, SourcesMixin):
     class Meta:
         proxy = True
 
+    @classmethod
+    def most_recent_past_meetings(cls):
+        '''
+        Returns meetings in the current month that have occured in the past
+        two weeks.
+        '''
+        current_month = timezone.now().month
+        two_weeks_ago = timezone.now() - timedelta(weeks=2)
+
+        meetings_in_past_two_weeks = cls.objects.filter(
+            Q(start_time__month=current_month), Q(start_time__gte=(two_weeks_ago)), Q())
```

Comment:
Since you're not doing any fancy logic here (using OR, or programmatically building filters in a loop), you don't need to use `Q()` instances here – you can pass your keyword arguments directly to `filter()`.